### PR TITLE
Super Fuel Recipe Changes

### PR DESCRIPTION
**Pull request by Steelux8**
- Removed recipes that used McGuffium, a fluid that is currently uncraftable;
- Removed recipes with LPG and Heavy Fuel inputs, as they are much easier to make than CBD. It makes little sense to have cheap oil-related fuels take the place of CBD, and this process can work as an upgrade to CBD on the advanced Large Boilers;
- Removed some recipes on the magic fuel side that used Mercury + Thaumcraft inputs, keeping only the basic, inefficient Mercury options with Blaze Powder and Ender Eyes,
- Changed all recipes with tiny dusts to use full dusts instead, and changed ratios on most of them, also for balancing purposes.

The changed list of recipes is essentially as follows:
- Fuel Binder is made with cheap glue or super glue, plus some cheap (Sulfur, Wood, Lithium, Cesium) or expensive (Gelled Toluene, Toluene, Tiny Pile of PBI) for different amounts of binder outputs;
- Magic Fuel Binder takes the regular one plus cheap vanilla materials + mercury (inefficient), or Thaumcraft shard dusts alongside either Fiery Steel (automated from Salis Mundus) or Liquid Death;
- Both regular and magic Super Fuel blocks use their respective binders + CBD + a block of Lignite Coal/Charcoal/Coal/Coal Coke, more efficient on the CBD usage as the fuel value of the block goes up.

Balance feedback is welcome.

### CODE REVIEW

File: src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java [hunk 1]
```diff
@@ -152,86 +152,43 @@ public void run() {
...
-        GT_Values.RA.addMixerRecipe(ItemList.SFMixture.get(2), GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.InfusedOrder, 1L), GT_Values.NI, GT_Values.NI, GT_Values.NI, GT_Utility.getIntegratedCircuit(1), Materials.Mercury.getFluid(50), null, ItemList.MSFMixture.get(2), 100, 64);
-        GT_Values.RA.addMixerRecipe(ItemList.SFMixture.get(2), GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.InfusedWater, 1L), GT_Values.NI, GT_Values.NI, GT_Values.NI, GT_Utility.getIntegratedCircuit(1), Materials.Mercury.getFluid(50), null, ItemList.MSFMixture.get(2), 100, 64);
-
-            FluidStack tFD = FluidRegistry.getFluidStack("fluiddeath", 10);
+        GT_Values.RA.addMixerRecipe(ItemList.SFMixture.get(20), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.InfusedAir, 1L), GT_Values.NI, GT_Values.NI, GT_Values.NI, GT_Utility.getIntegratedCircuit(1), Materials.FierySteel.getFluid(50), null, ItemList.MSFMixture.get(20), 200, 64);
```
Comment (Dream-Master) [inline src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java:167]: Not sure if 50 is a good value? Fiery Blood glass bottle has voulume of 250. smelted parts use 9/18/36/72/144 etc

**Comment by ScriptedPiky** [comment]
McGuffium I believe has a recipe, it's just that it's intentionally not shown in NEI if I remember correctly, along with some other obscure GT potions/recipes

**Comment by Steelux8** [comment]
> McGuffium I believe has a recipe, it's just that it's intentionally not shown in NEI if I remember correctly, along with some other obscure GT potions/recipes

If this is true, I think it's one of the worst design choices I've ever seen in modded MC, in my opinion.

**Review by Dream-Master — APPROVED** [review]

**Comment by C0bra5** [comment]
> McGuffium I believe has a recipe, it's just that it's intentionally not shown in NEI if I remember correctly, along with some other obscure GT potions/recipes

as far as i can tell this claim appears to be incorrect: https://github.com/GTNewHorizons/GT5-Unofficial/pull/2704#issuecomment-2208035737

**Pull request merged**